### DISCUSSION TITLE
Emby login?

### DISCUSSION THREAD

**Discussion by havrancek** [seerr-team/seerr#172] [Ideas]
I would like to have an option to login through Emby. Is that possible? I don't have Plex nor Jellyfin and I would like to use it with my Emby instance. Thanks for this.

**Comment by sambartik** [comment]
Instructions are available over at https://hub.docker.com/r/fallenbagel/jellyseerr

**Comment by fallenbagel** [comment]
Yes as sambartik said. What's important is the environment variable as mentioned in the dockerhub instructions

**Comment by Riddler84** [comment]
Same problem here. I installed it on Windows, therefore I can't use that Docker variable. I mean, it works, but all the labels saying Jellyfin and play buttons don't work because of the wrong url.

Where can I set this variable?

**Reply by fallenbagel** [reply]
You create a file called `.env` in the root folder and add the variable

**Reply by fallenbagel** [reply]
Remember to do this while you have jellyseerr stopped
